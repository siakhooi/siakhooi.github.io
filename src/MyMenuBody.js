import React from 'react'
import PropTypes from 'prop-types'

MyMenuBody.propTypes = {
  children: PropTypes.array
}

function MyMenuBody (props) {
  return (
    <div className="MyMenuBody">{props.children}</div>
  )
}

export default MyMenuBody
