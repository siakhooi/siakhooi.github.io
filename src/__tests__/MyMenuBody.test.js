import React from 'react'
import renderer from 'react-test-renderer'
import MyMenuBody from '../MyMenuBody'

it('MyMenuBody/1', () => {
  const component =
    renderer.create(
      <MyMenuBody><div>ABC</div><div>ABC</div></MyMenuBody>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
