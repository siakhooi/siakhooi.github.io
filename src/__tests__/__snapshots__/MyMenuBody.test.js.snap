// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyMenuBody/1 1`] = `
<div
  className="MyMenuBody"
>
  <div>
    ABC
  </div>
  <div>
    ABC
  </div>
</div>
`;
